### PR TITLE
Device: Dlink dap2660 add processors and mempools

### PR DESCRIPTION
**Pull request by tomarch**
Add processors and mempools.
Also made some enhancement for version and hardware.
test on DAP-2660 and DAP-2360
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/polling/mempools/dlinkap.inc.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2016 Søren Friis Rosiak <sorenrosiak@gmail.com>
```
Comment (laf) [inline includes/polling/mempools/dlinkap.inc.php]: You've re-used someone else's copyright here.

File: includes/polling/os/dlinkap.inc.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?php
+
```
Comment (laf) [inline includes/polling/os/dlinkap.inc.php]: Please add your copyright here.

File: includes/polling/processors/dlinkap.inc.php [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?php
```
Comment (laf) [inline includes/polling/processors/dlinkap.inc.php]: You shouldn't need this file. You're registering the oid already in discovery.

**Comment by tomarch** [comment]
Thanks for your review.
I've done requested changes.

**Pull request merged**